### ISSUE TITLE
logtank_topic_v2:  Error 415 Unsupported Media Type

### ISSUE THREAD

**Issue by rramge** [opentelekomcloud/terraform-provider-opentelekomcloud#1928]
# Terraform provider version

Terraform v1.3.0-alpha20220817

on linux_amd64

+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.31.1

# Affected Resource(s)

module.logtank.opentelekomcloud_logtank_topic_v2

# Terraform Configuration Files
```hcl
variable "group_name" {
  type        = string
  default     = null
  description = "(Required) Specifies the log group name."
}

variable "topic_names" {
  type        = list(string)
  default     = null
  description = "A list of topic names"
}
```

```hcl
resource "opentelekomcloud_logtank_group_v2" "this" {

  group_name = var.group_name

}

resource "opentelekomcloud_logtank_topic_v2" "this" {
  count = length(var.topic_names)

  group_id   = opentelekomcloud_logtank_group_v2.this.id
  topic_name = var.topic_names[count.index]
}
```

```hcl
module "logtank" {
  source = "../../modules/terraform-opentelekomcloud-lts"

  group_name = "lts-demo"
  topic_names = ["topic-01", "topic-02", "topic-03", "topic-04", "topic-05" ]
}
```

# Debug Output/Panic Output

```
╷
│ Error: Error deleting log topic: Expected HTTP response code [202 204] when accessing [DELETE https://lts.eu-de.otc.t-systems.com/v2.0/c7b5384e487149d696b7cfc420589afe/log-groups/45f256e2-cf1d-488e-ae9b-eca7e901c2de/log-topics/7b1d86ed-1c1e-4073-abee-6019fc22cb26], but got 415 instead
│ {"timestamp":1662975277180,"status":415,"error":"Unsupported Media Type","message":"Content type '' not supported","path":"/v2.0/c7b5384e487149d696b7cfc420589afe/log-groups/45f256e2-cf1d-488e-ae9b-eca7e901c2de/log-topics/7b1d86ed-1c1e-4073-abee-6019fc22cb26"}
│ 
│ 
╵
```

# Steps to Reproduce

1. `terraform apply`
2. `terraform destroy`

# Expected Behavior

All resources get destroyed.

# Actual Behavior

Error message from above, no resources get destroyed.

# Important Factoids

OTC region eu-de

# References

**Comment by vladimirvshivkov** [comment]
The issue was fixed in #1908 and prepared to release on wednesday.

**Comment by rramge** [comment]
Okay, bad timing from my side, didn't see it :-) Thanks!
